src/06-longitudinal-and-multi-site-studies.md

```diff
@@ -1,21 +1,9 @@
 # Longitudinal and multi-site studies
 
 Multiple sessions (visits) are encoded by adding an extra layer of directories
-and file names in the form of `ses-<label>`.
-Session labels MUST consist only of alphanumeric characters `[a-zA-Z0-9]`
-and SHOULD be consistent across subjects.
-If numbers are used in session labels we RECOMMEND using zero padding
-(for example `ses-01`, `ses-11` instead of `ses-1`, `ses-11`). This makes
-results of alphabetical sorting more intuitive. Acquisition time of session can
-be defined in the [sessions file](#sessions-file).
```

Comment:
Is this recommendation being dropped altogether? Or is it duplicating something written elsewhere?

Reply:
yup this is dropped because it is already mentioned in the common principles: see earlier discussions.

https://github.com/bids-standard/bids-specification/pull/805#discussion_r644656450